gibbon-compiler/src/Gibbon/Common.hs

```diff
@@ -491,3 +501,10 @@ truePrinted = "#t"
 
 falsePrinted :: String
 falsePrinted = "#f"
+
+unwrapLocVar :: LocVar -> Var
+unwrapLocVar locvar = case locvar of 
+                            Single loc -> loc
```

Comment:
This may be not very future-proof: will it always be possible to turn a LocVar into a Var? If there's a chance that the answer is no, this function may be more honest by returning `Maybe Var`. Of course, this could make its clients unhappy. In any case, it'd be helpful to add a comment on why it's possible to do what it does (turn one kind of var into another kind of var).

Reply:
No this is not at all future proof, just a hack for now. 
There should be a new like there that errors out on any other value of LocVar. 
But for now its fine since we just have a single value for LocVar.

Reply:
I'd still leave a comment explaining why it's possible to convert between the kinds of variables.